common/main/en.xml

```diff
@@ -139,6 +139,7 @@ annotations.
 			<language type="cop">Coptic</language>
 			<language type="cps">Capiznon</language>
 			<language type="cr">Cree</language>
+			<language type="cr" alt="long">Woods Cree</language>
```

Comment:
I don't think this is a great situation. Woods Cree isn't a "longer" form of Cree. 

Content (whether a document, an audio recording, etc.) isn't going to be just in Cree, it's going to be in Woods Cree, or Plains Cree, or Swampy Cree, etc.  Perhaps the macrolanguage isn't helpful here and should be removed. (The pronunciation and orthography is distinct between these three, in both Latn and Cans scripts.)

For now, I kind of think the best and most consistent change would be `<language type="cr">Woods Cree</language>`

Reply:
```suggestion
			<language type="cr">Woods Cree</language>
```

Reply:
That would overturn from our long-standing policy for dealing with the compatibility mess that macrolangauges created.*

We **don't** say 
- Tosk Albanian, but rather Albanian
- Standard Arabic, but rather Arabic
- Standard Estonian, but rather Estonian
- Eastern Yiddish, but rather Yiddish
- Northern Uzbek, but rather Uzbek
- Iranian Persian, but rather Persian (or Farsi)

and so on. 

That is, we use the code and name of the macrolanguage for the predominant encompassed code. The one exception off the top of my head is for "zh", where we have:

```
<language type="zh">Chinese</language>
<language type="zh" alt="long">Mandarin Chinese</language>
```

So that's why I _was_ "consistent" in following that model for Cree. 

* If the macrolanguage policy had been _consistently_ applied by ISO, they'd have made German (de) as a macrolanguage, and given a new code for Standard German (eg, dex?) as an encompassed language (and made the 30ish other Germans like Colonia Tovar German, Swiss German, Low German, and so on also be encompassed languages). And the same for many other languages (French, Hindi, ...).

Reply:
Would there be some option to have CLDR decide that cr isn't (Or isn't considered) a macrolanguage? And just allow `cwd` to stand on its own? 

I hear you about consistency but it does not work with the identity of the languages encompassed under cr.

The difference with German is that people do refer to `de` as Deutsch whereas even the name of the language differs between cwd/crk/csw - nehenawewin, neheyawewin and nehethawewin (not respectively).

Reply:
I'm fine to land and then discuss. And I know this is long standing policy and not new.

Reply:
> Those appear to be just the same word with slightly different pronunciations in different dialects, according to https://en.wikipedia.org/wiki/Cree_language#Dialect_criteria, which is exposed because written Cree is more phonetic than English.
> 
> 
> 
> It's as if we said that there were 3 different names for English: /ˈɪŋɡlɪʃ/ (UK), /ˈɪŋlɪʃ/ (US), /ˈɪŋɡləʃ/ (AU).
> 
> 
> 
> (They also appear to be really different dialects, not different languages, at a least according to https://en.wikipedia.org/wiki/East_Cree)
> 
> 
> 
> We should take this up in further in our next meeting: I filed https://unicode-org.atlassian.net/browse/CLDR-18142 about this.
> 
> 
> 
> In the meantime, I'll put out a call for taking the ticket for 47. We can always reverse the change for cr should TC decide it.

Right but one is not more general or more representative than the other. There are vocabulary differences as well.

Reply:
Generality, representativity, vocabulary differences, are not relevant to choosing the predominant encompassed language.

As I said, let's discuss in the meeting.

Reply:
As you mentioned, this change isn't about Macrolanguages -- it's about validity. So I'll accept to unblock. We probably don't have enough time to properly discuss the issue before the new year anyway, so let's not get ahead of ourselves.

We should follow up in a separate ticket -- but decide the timeline for this.